.github/workflows/main.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         build_type: [static_build, shared_build]
 
     steps:
+      - name: Checkout
       - uses: actions/checkout@v4
```

Comment:
I think you have an extra `-` here, which prevents the workflow to run

Reply:
Good eye